.circleci/config.yml

```diff
@@ -0,0 +1,93 @@
+---
+version: 2
+
+jobs:
+  build-tutorials:
+    docker:
+      - image: fpco/stack-build:lts-9.21
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+            - v1-stack-{{ checksum "stack.yaml" }}
+            - v1-stack-
+
+      - run:
+          name: Upgrading stack
+          command: |
+            echo 'export PATH=$HOME/.local/bin:$PATH' >> $BASH_ENV
+            stack upgrade
+
+      - run:
+          name: Installing compiler
+          command: stack setup
+
+      - restore_cache:
+          keys:
+            - v1-stack-work-{{ checksum "stack.yaml" }}
+            - v1-stack-work-
+
+      - run:
+          name: Installing dependencies
+          command: stack build --only-dependencies
+
+      - save_cache:
+          key: v1-stack-{{ checksum "stack.yaml" }}
+          paths:
+            - ~/.stack
+
+      - run:
+          name: Compiling
+          command: stack build
+
+      - save_cache:
+          key: v1-stack-work-{{ checksum "stack.yaml" }}
+          paths:
+            - .stack-work
+
+      - run:
+          name: Build tutorials site
+          command: make build
+
+  deploy-master:
+    docker:
+      - image: stackbuilders/hapistrano:latest
+    steps:
+      - checkout
+      - run:
+          name: Deploying
+          command: |
+            echo -e "Host * \n  StrictHostKeyChecking no" >> ~/.ssh/config
```

Comment:
I know this gets merged already but what you think in add something like: 
```
- run: 
    name: SSH keys
      command: |
        echo -e "$KNOWN_HOSTS" >> ~/.ssh/known_hosts
        echo -e "Host * \n  ControlMaster yes \n  ControlPersist 10s" >> ~/.ssh/config
- add_ssh_keys:
    fingerprints:
      - "yy:yy:yy:yy:xx:xx:xx:e1:d2:aa:e1:aa:5b:aa:c6:45"```
Instead of just accept all new ssh connections? 
I think this is better from a security stand point and is standard way of doing it in circleCI 2.0